packages/blocks/src/store/test/selectors.js

```diff
@@ -86,72 +86,74 @@ describe( 'selectors', () => {
 
 	describe( 'getChildBlockNames', () => {
 		it( 'should return an empty array if state is empty', () => {
-			const state = {};
+			const state = {
+				blockTypes: {},
+			};
 
 			expect( getChildBlockNames( state, 'parent1' ) ).toHaveLength( 0 );
 		} );
 
 		it( 'should return an empty array if no children exist', () => {
 			const state = {
-				blockTypes: [
-					{
+				blockTypes: {
+					child1: {
```

Comment:
These fixtures were incorrect, the structure is not an array, instead it is an object keyed by block type name:

https://github.com/WordPress/gutenberg/blob/3284f343a4c69a74a1e29eac03a14d0d2c6c9cc1/packages/blocks/src/store/reducer.js#L98

Those tests were passing because `Object.values( [ 1, 2, 3 ] )` and `Object.values( { test: 1, foo: 2, bar: 3 } )` will yield two identical arrays.

We're using the opportunity to fix them anyways for better test precision and to represent the actual data format.